tests/util/test_yaml.py

```diff
@@ -153,13 +169,18 @@ def test_include_dir_merge_list(self, mock_walk):
     def test_include_dir_merge_list_recursive(self, mock_walk):
         """Test include dir merge list yaml."""
         mock_walk.return_value = [
-            ['/tmp', ['tmp2'], ['first.yaml']],
+            ['/tmp', ['tmp2', '.ignore', 'ignore'], ['first.yaml']],
             ['/tmp/tmp2', [], ['second.yaml', 'third.yaml']],
+            ['/tmp/.ignore', [], ['ignore.yaml']],
+            ['/tmp/ignore', [], ['.ignore.yaml']]
         ]
 
         with patch_yaml_files({
-                '/tmp/first.yaml': '- one', '/tmp/tmp2/second.yaml': '- two',
-                '/tmp/tmp2/third.yaml': '- three\n- four'
+            '/tmp/first.yaml': '- one',
+            '/tmp/tmp2/second.yaml': '- two',
+            '/tmp/tmp2/third.yaml': '- three\n- four',
+            '/tmp/.ignore/ignore.yaml': '- ignored1',
```

Comment:
There should be no need to patch these yaml files. Not patching them should raise FileNotFound exception if you try to read them (which shows your os.walk successful excluded the dot files)